src/main/java/org/snomed/snowstorm/core/data/domain/Axiom.java

```diff
@@ -119,12 +120,49 @@ public Axiom setRelationships(Set<Relationship> relationships) {
 		return this;
 	}
 
-	@JsonIgnore
 	public ReferenceSetMember getReferenceSetMember() {
 		return referenceSetMember;
 	}
 
 	public void setReferenceSetMember(ReferenceSetMember referenceSetMember) {
 		this.referenceSetMember = referenceSetMember;
 	}
+
+    public void clone(Axiom axiom) {
+        setAxiomId(axiom.getAxiomId());
+        setModuleId(axiom.getModuleId());
+        setActive(axiom.isActive());
+        setReleased(axiom.isReleased());
+        setDefinitionStatusId(axiom.getDefinitionStatusId());
```

Comment:
I've added a clone method to all of the components. These clone methods are used so that the "auto-merged version" of the Concept is based on the LHS of the conflict window.